### PR TITLE
Manage weekly seminars as markdown files instead of using a remote ICS

### PR DESCRIPTION
**Pull request by ArtemisLemon**
Changed how the seminar page and ics work, from parsing an ics hosted by a third party, to generating from local data the page and the ics

current implementation has a "description" field in the Yaml header of the .md of each post (the most robust version right now can take any characters except "), and additionally uses the body of the page to hold similar (copy?) information
- when getting the description of each event, the oldICS seems to have given me valid markdown (specifically for the links)
- this description was put in both the body of the post and a "description" field of the yaml header
- post.content returns an html transformation of the md, if you use the strip_html filter it removes link titles and urls
- post.description returns the exact field (with " replaced with ', done in the python script)

### CODE REVIEW

File: seminar/2223/_posts/NaomodWeeklySeminars
Comment (ebousse) [inline seminar/2223/_posts/NaomodWeeklySeminars:1]: Isn't this a generated file? Should it be committed?

**Review by ebousse — COMMENTED** [review]
Thanks, looks great!

Still confused about the `description` field though, why keep it if it contains the same as the body?

File: seminar/2223/_posts/NaomodWeeklySeminars
Comment (ArtemisLemon) [reply seminar/2223/_posts/NaomodWeeklySeminars:1]: That's the original ics, which the ics2md.py uses, the generated ics is naomodWeekly.ics in the root folder

Comment (ebousse) [reply seminar/2223/_posts/NaomodWeeklySeminars:1]: Ah I see, so it's the raw data used to produce the first set of YAML files. Makes sense.

**Comment by ArtemisLemon** [comment]
> Still confused about the `description` field though, why keep it if it contains the same as the body?

this is the part I'm struggling to clarify the most:

in the liquid bits post.description gives me exactly what's written in the field, most notably [title](url) doesn't get touched. A [title](url) in the post.md body causes post.content to return  <a href=url>title</a>.

If I only had the description in the post body, either the ics description would have raw html, or (using the strip_html filter in liquid: {% post.content | strip_html %} ) have the links removed from the description.

**Pull request merged**